modules/cluster-logging-collector-limits.adoc

```diff
@@ -32,7 +32,7 @@ metadata:
 spec:
   collection:
     logs:
-      fluentd:
+      collector:
```

Comment:
collector is not a  valid  properties. @jcantrill  Do we need a bug?
https://github.com/openshift/cluster-logging-operator/blob/master/bundle/manifests/logging.openshift.io_clusterloggings.yaml#L54